src/site/content/en/lighthouse-accessibility/use-landmarks/index.md

```diff
@@ -13,7 +15,7 @@ HTML5 elements such as `main`, `nav`, and `aside` act as landmarks,
 or special regions on the page to which a screen reader can jump.
 By using landmark elements,
 you can dramatically improve the navigation experience on your site
-for users of assitive technology.
+for users of assistive technology.
 Learn more in
 [HTML 5 and ARIA Landmarks](https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/landmarks.html).
 
```

Comment:
What do you think about the example below? It's using `role` (I'm assuming as a fallback for missing landmark roles) but I'm not sure if it actually needs to do that. I'm pretty sure modern browsers support the landmark elements in that example.

Reply:
Yeah, that's weird. Removing the roles.